unison-cli/transcripts/Transcripts.hs

```diff
@@ -60,31 +61,31 @@ testBuilder' ucm dir prelude transcript = scope transcript $ do
     dropRunMessage :: String -> String
     dropRunMessage = unlines . reverse . drop 3 . reverse . lines
 
-
 buildTests :: TestConfig -> TestBuilder -> FilePath -> Test ()
 buildTests config testBuilder dir = do
   io
-     . putStrLn
-     . unlines
-     $ [ ""
-       , "Searching for transcripts to run in: " ++ dir
-       ]
+    . putStrLn
+    . unlines
+    $ [ "",
+        "Searching for transcripts to run in: " ++ dir
+      ]
   files <- io $ listDirectory dir
-  let
-    -- Any files that start with _ are treated as prelude
-    (prelude, transcripts) =
-      partition ((isPrefixOf "_") . snd . splitFileName)
-      . sort
-        -- if there is a matchPrefix set, check for a prefix match - or return True
-      . filter (\f -> maybe True (`isPrefixOf` f) (matchPrefix config))
-      . filter (\f -> takeExtensions f == ".md") $ files
+  -- Any files that start with _ are treated as prelude
+  let (prelude, transcripts) =
```

Comment:
Here's the block that actually changed, it partitions before the filtering step now.

Reply:
Struggling a little to read the diff while converting between `.` and `&` in my head but I guess this looks right. 

Am I understanding correctly that the only real change in this PR was the addition of `second ( ... )`?